### ISSUE TITLE
Doc examples use `xrft.dft` even though it's deprecated

### ISSUE THREAD

**Issue by tomchor** [xgcm/xrft#199]
Hi, everyone, I noticed all 4 [doc examples](https://xrft.readthedocs.io/en/latest/DFT-iDFT_example.html#Example-of-discrete-and-inverse-discrete-Fourier-transform) use `xrft.dft` for the forward fft, but when I try to get info on the function the docstring claims it's deprecated:

```ipython
In [22]: xrft.dft?
Signature: xrft.dft(da, dim=None, true_phase=False, true_amplitude=False, **kwargs)
Docstring: Deprecated function. See fft doc
File:      ~/miniconda3/envs/py310/lib/python3.10/site-packages/xrft/xrft.py
Type:      function
```

All the snippets I tried worked the same when swapping `xrft.dft` for `xrft.fft`, so I guess that's the new function to use? I figure it'd be best to open an issue first to make sure, before opening a PR to fix this.

I'm using version `1.0.1` btw.

**Comment by roxyboy** [comment]
Yes, the documentation examples are a bit out of date... `xrft.fft` is the new function to use. `xrft.dft` and `xrft.fft` have different default input arguments.
